### PR TITLE
update to troubleshoot v0.9.29

### PR DESCRIPTION
**Pull request by laverya**

### CODE REVIEW

File: pkg/redact/redact.go [hunk 1]
```diff
@@ -42,6 +44,28 @@ func GetRedactSpec() (string, string, error) {
...
+func GetRedact() (*v1beta1.Redactor, error) {
+	spec, _, err := GetRedactSpec()
+	if err != nil {
+		return nil, err
+	}
+	if spec == "" {
+		return nil, nil
+	}
+
+	scheme.AddToScheme(scheme.Scheme)
```
Comment (emosbaugh) [inline pkg/redact/redact.go:56]: can you move this to an init function?

File: pkg/handlers/troubleshoot.go [hunk 2]
```diff
@@ -81,7 +80,7 @@ func GetTroubleshoot(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	troubleshootclientsetscheme.AddToScheme(scheme.Scheme)
+	scheme.AddToScheme(scheme.Scheme)
```
Comment (emosbaugh) [inline pkg/handlers/troubleshoot.go:83]: move this to an init function

File: pkg/redact/redact.go (hunk 1 above)
Comment (laverya) [reply pkg/redact/redact.go:56]: do we do this in an init function anywhere else?

Comment (emosbaugh) [reply pkg/redact/redact.go:56]: we should. ive mentioned it before. this is how they do it in the k8s codebase
https://github.com/kubernetes/kubernetes/blob/0e7376d9cb1ec74bcb3c10a664a330bc9d60df64/staging/src/k8s.io/metrics/pkg/client/clientset/versioned/scheme/register.go#L57

Comment (emosbaugh) [reply pkg/redact/redact.go:56]: https://godoc.org/sigs.k8s.io/controller-runtime/pkg/scheme

Comment (emosbaugh) [reply pkg/redact/redact.go:56]: in any case it cant be in concurrent code. we could add it to the main or to an initialization function

Comment (laverya) [reply pkg/redact/redact.go:56]: main sounds like a better location then
to make sure multiple inits don't run at once

Comment (laverya) [reply pkg/redact/redact.go:56]: though this _is_ a different scheme package

**Pull request merged**